ckanext/validation/jobs.py

```diff
@@ -54,54 +55,62 @@ def run_validation_job(resource):
         {'ignore_auth': True}, {'id': resource['package_id']})
 
     source = None
-    if resource.get(u'url_type') == u'upload':
+    if resource.get('url_type') == 'upload':
         upload = uploader.get_resource_uploader(resource)
         if isinstance(upload, uploader.ResourceUpload):
-            source = upload.get_path(resource[u'id'])
+            source = upload.get_path(resource['id'])
         else:
             # Upload is not the default implementation (ie it's a cloud storage
             # implementation)
             pass_auth_header = t.asbool(
-                t.config.get(u'ckanext.validation.pass_auth_header', True))
-            if dataset[u'private'] and pass_auth_header:
+                t.config.get('ckanext.validation.pass_auth_header', True))
+            if dataset['private'] and pass_auth_header:
                 s = requests.Session()
                 s.headers.update({
-                    u'Authorization': t.config.get(
-                        u'ckanext.validation.pass_auth_header_value',
+                    'Authorization': t.config.get(
+                        'ckanext.validation.pass_auth_header_value',
                         _get_site_user_api_key())
                 })
 
-                options[u'http_session'] = s
+                options['http_session'] = s
 
     if not source:
-        source = resource[u'url']
-
-    schema = resource.get(u'schema')
-    if schema and isinstance(schema, str):
-        if schema.startswith('http'):
-            r = requests.get(schema)
-            schema = r.json()
+        source = resource['url']
+
+    schema = resource.get('schema')
+    if schema:
+        if isinstance(schema, str):
+            if schema.startswith('http'):
```

Comment:
So, if `schema` is a string that doesn't start with "http" then it's preserved unchanged?